llms/mlx_lm/convert.py

```diff
@@ -60,39 +60,30 @@ def fetch_from_hub(
 ) -> Tuple[Dict, dict, transformers.PreTrainedTokenizer]:
     model_path = get_model_path(model_path)
 
-    weight_files = glob.glob(f"{model_path}/*.safetensors")
```

Comment:
The weights directly loaded from safetensors were not sanitized, so it will still throw an error during quantization. Therefore, change to load the model from utils which has weight sanitization.

Reply:
Good catch!